dracut.sh

```diff
@@ -2598,14 +2598,18 @@ umask 077
 if [[ $uefi == yes ]]; then
     if [[ $kernel_cmdline ]]; then
         echo -n "$kernel_cmdline" > "$uefi_outdir/cmdline.txt"
-    elif [[ $hostonly_cmdline == yes ]] && [ -d "$initdir/etc/cmdline.d" ]; then
-        for conf in "$initdir"/etc/cmdline.d/*.conf; do
-            [ -e "$conf" ] || continue
-            printf "%s " "$(< "$conf")" >> "$uefi_outdir/cmdline.txt"
-        done
+    elif [[ $hostonly_cmdline == yes ]]; then
+        if [ -d "$initdir/etc/cmdline.d" ]; then
+            for conf in "$initdir"/etc/cmdline.d/*.conf; do
+                [ -e "$conf" ] || continue
+                printf "%s " "$(< "$conf")" >> "$uefi_outdir/cmdline.txt"
+            done
+        elif [ -e "$initdir/proc/cmdline" ]; then
```

Comment:
Is the running system `/proc` mounted in the temporary directory where the initrd is being created?

Reply:
The `initdir` is a temporary directory where dracut adds stuff before packing its content into an initramfs image.

https://github.com/dracutdevs/dracut/blob/4012314f204806a67d038f378f525646814d12ea/dracut.sh#L1258-L1259

So, `/var/tmp/dracut.??????/initramfs/proc/cmdline` should never exist while dracut is running.